docs/services/slam/orbslamv3/_index.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "ORB-SLAM3 Integrated Library"
+linkTitle: "ORB-SLAM3"
+weight: 70
+type: "docs"
+draft: true
```

Comment:
This is now a draft per. update from @kkufieta!

Reply:
@EshaMaharishi raised the question whether or not we should simply delete the orbslam documentation, since the content would still be preserved in the git history. 

How does the docs team go about things like these?

Reply:
Thanks @kkufieta ! Yes, I was wondering if we can simply remove it, since it will remain in the git history?